src/tools/cephfs/top/cephfs-top

```diff
@@ -184,6 +186,14 @@ class FSTopBase(object):
                 elif item == FS_TOP_MAIN_WINDOW_COL_MNT_ROOT:
                     client_id_meta.update({item:
                                            client_meta[CLIENT_METADATA_MOUNT_ROOT_KEY]})
+                elif item == FS_TOP_MAIN_WINDOW_COL_MNTPT_TYPE:
+                    if "kernel_version" in client_meta:
```

Comment:
Older kernel do not advertise `kernel_version` in client metadata, so with this logic they would get reported as `libcephfs` clients :/

Reply:
Aren't we giving the number of kernel clients with this same way? https://github.com/ceph/ceph/blob/main/src/tools/cephfs/top/cephfs-top#L268 .

Reply:
How old are the kernels? Does libcephfs have any of its own special metadata, or formatting we can use, to disambiguate?

Reply:
@neesingh-rh Have you checked regarding @gregsfortytwo comment? cephfs-top might be reporting incorrect libcephfs clients if older kernel clients are using the file system.

Reply:
@neesingh-rh PTAL.